mysql-test/main/backup_interaction.test

```diff
@@ -221,8 +221,9 @@ drop table t1;
 --echo # OTOH we most probably allow them under FTWRL as well
 --echo #
 --disable_view_protocol
-CREATE TABLE t1 (col1 INT) ENGINE = InnoDB;
+CREATE TABLE t1 (col1 INT)stats_persistent=0, ENGINE = InnoDB;
 insert into t1 values (1);
+--source ../suite/innodb/include/wait_all_purged.inc
```

Comment:
In 10.4 and 10.5, this file must not be included without also setting `innodb_purge_rseg_truncate_frequency=1`. Otherwise, it will slow down the test extremely. I checked this on the mandatory [amd64-ubuntu-2004-debug](https://buildbot.mariadb.org/#/builders/369) builder. It would be this fast before this change:
```
main.backup_interaction 'innodb'         w18 [ pass ]     99
```
After this change:
```
main.backup_interaction 'innodb'         w19 [ pass ]   7955
```